core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -135,3 +124,24 @@
     width: 220px;
   }
 </style>
+
+<link rel="stylesheet" href="/third_party/static/perfectscrollbar-0.6.16/css/perfect-scrollbar.css" />
```

Comment:
@seanlip please check if you meant the new changed. i was bit confused about "end -- should be".